docs/basic-features/data-fetching/client-side.md

```diff
@@ -0,0 +1,68 @@
+---
+description: 'Learn how to use SWR, a data fetching React hook that handles caching, revalidation, focus tracking, refetching on interval and more.'
+---
+
+# Client-side data fetching
+
+Client-side data fetching is useful when your page doesn't require SEO indexing, when you don't need to pre-render your data, or when your pages content needs to update frequently. Unlike the server-side rendering APIs, you can use client side data fetching at the component level.
+
+If done at the page level, the data is fetched at runtime, and the contents of the page updated as the data changes. When used at the component level, the data is fetched at the time of the component mount, and the contents of the component are updated as the data changes.
+
+It's important to note that using client-side data fetching can affect the performance of your application and the load speed of your pages. This is because the data fetching is done at the time of the component or pages mount, and the data is not cached.
+
+## Client-side data fetching with useEffect
```

Comment:
This section contradicts the page description:

https://github.com/vercel/next.js/pull/30615/files#r764853007